user-integration-guide/symfony-integration.md

```diff
@@ -0,0 +1,218 @@
+# Integrating Opentelemetry PHP into Symfony Applications
+
+## Introduction 
+
+As a developer, you might be wondering how  OpenTelemetry could be beneficial to you. You are not alone.  Without practical examples, the usefulness of distributed tracing can be difficult to grasp for persons without a cloud or site reliability engineering background. This user guide shows how OpenTelemetry could be useful to gain insights into exceptions happening within an application. This example uses the OpenTelemtry PHP library integrated into a Symfony application, bundled with Jaeger and Zipkin, for visualizing data.  
+## Prerequisites
+To follow this guide you will need:
+
+* PHP Installed, this example uses PHP 7.4 .
+* [Composer](https://getcomposer.org/download/ ) for dependency management. 
+* [Symfony CLI](https://symfony.com/download) for managing your Symfony application
+*  [Docker](https://docs.docker.com/get-docker/) for bundling our visualization tools
```

Comment:
The spacing for `Docker` is indented too far.  You could also link to the part of the `README.md` that has the installation instructions here.